source/extensions/quic_listeners/quiche/platform/BUILD

```diff
@@ -97,21 +97,29 @@ envoy_cc_library(
     srcs = [
         "quic_cert_utils_impl.cc",
         "quic_text_utils_impl.cc",
-    ],
+    ] + envoy_select_quiche([
```

Comment:
Also I had no idea what this was, and while Bin clued me in offline, would you mind adding a comment to envoy_build_system.bzl at the envoy_select_quiche definition explaining when we use this just in case anyone else gets confused?  :-)

Reply:
Sure, added a comment in envoy_build_system.bzl.